src/test/java/uk/gov/homeoffice/digital/sas/timecard/listeners/TimeEntryKafkaEntityListenerTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.homeoffice.digital.sas.timecard.listeners;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.homeoffice.digital.sas.timecard.enums.KafkaAction;
+import uk.gov.homeoffice.digital.sas.timecard.kafka.producers.KafkaProducerService;
+import uk.gov.homeoffice.digital.sas.timecard.model.TimeEntry;
+
+@ExtendWith(MockitoExtension.class)
+class TimeEntryKafkaEntityListenerTest {
+
+  private final UUID ownerId = UUID.randomUUID();
```

Comment:
nit: this can be static and the use the conventional capitalised naming

Reply:
Updated